src/main/java/io/github/fusionflux/portalcubed/data/tags/PortalCubedTags.java

```diff
@@ -0,0 +1,35 @@
+package io.github.fusionflux.portalcubed.data.tags;
+
+import io.github.fusionflux.portalcubed.PortalCubed;
+import net.minecraft.core.registries.Registries;
+import net.minecraft.tags.TagKey;
+import net.minecraft.world.entity.EntityType;
+
+public class PortalCubedTags {
```

Comment:
Using subclasses like this is bad, make separate classes. There's also nothing to init, don't do that

Reply:
init is for classloading

Reply:
but there's no reason to classload it. Nothing is registered, just led it load whenever it's first referenced.